test/lib/ufe/testParentCmd.py

```diff
@@ -386,6 +386,277 @@ def testParentAbsoluteSingleMatrixOp(self):
+                    ufe.PathString.string(xformPath))
+
+        def checkParentDone():
+            # The xform now has the capsule as its child.
+            xformChildren = xformHier.children()
+            self.assertEqual(len(xformChildren), 1)
+            self.assertIn('Capsule1', childrenNames(xformChildren))
+    
+            # Confirm that the capsule has not moved in world space.  Must
+            # re-create the prim and its scene item, as its path has changed.
+            capsulePath = ufe.Path(
+                [proxyShapePathSegment, 
+                 usdUtils.createUfePathSegment('/Xform1/Capsule1')])
+            capsulePrim = mayaUsd.ufe.ufePathToPrim(
+                ufe.PathString.string(capsulePath))
+            capsuleXformable = UsdGeom.Xformable(capsulePrim)
+            capsuleItem = ufe.Hierarchy.createItem(capsulePath)
+            capsuleT3d = ufe.Transform3d.transform3d(capsuleItem)
+            capsuleWorld = capsuleT3d.inclusiveMatrix()
+            assertVectorAlmostEqual(
+                self, capsuleWorldPre, matrixToList(capsuleWorld))
+    
+            # The capsule's transform ops now have Maya fallback stack ops.  A
+            # scale fallback op is added, even though it's uniform unit.
+            self.assertEqual(
+                capsuleXformable.GetXformOpOrderAttr().Get(), Vt.TokenArray((
+                    "xformOp:translate", "xformOp:rotateXYZ", "xformOp:rotateX",
+                    "xformOp:translate:maya_fallback", 
+                    "xformOp:rotateXYZ:maya_fallback",
+                    "xformOp:scale:maya_fallback")))
+
+        checkParentDone()
+
+        # Undo: the xform no longer has a child, the capsule is still where it
+        # has always been, and the fallback transform ops are gone.
+        cmds.undo()
+
+        xformChildren = xformHier.children()
+        self.assertEqual(len(xformChildren), 0)
+
+        capsulePath = ufe.Path(
+            [proxyShapePathSegment, usdUtils.createUfePathSegment('/Capsule1')])
+        capsulePrim = mayaUsd.ufe.ufePathToPrim(
+            ufe.PathString.string(capsulePath))
+        capsuleXformable = UsdGeom.Xformable(capsulePrim)
+        capsuleItem = ufe.Hierarchy.createItem(capsulePath)
+        capsuleT3d = ufe.Transform3d.transform3d(capsuleItem)
+        capsuleWorld = capsuleT3d.inclusiveMatrix()
+        assertVectorAlmostEqual(
+            self, capsuleWorldPre, matrixToList(capsuleWorld))
+        self.assertEqual(
+            capsuleXformable.GetXformOpOrderAttr().Get(), Vt.TokenArray((
+                "xformOp:translate", "xformOp:rotateXYZ", "xformOp:rotateX")))
+
+        # Redo: capsule still hasn't moved, Maya fallback ops are back.
+        cmds.redo()
+
+        checkParentDone()
+
+    @unittest.skipUnless(mayaUtils.previewReleaseVersion() >= 123, 'Requires Maya fixes only available in Maya Preview Release 123 or later.') 
+    def testZParentAbsoluteMultiMatrixOp(self):
```

Comment:
Evil test order dependence is back.  testParentAbsoluteMultiMatrixOp() causes testParentAbsoluteSingleMatrixOp() to fail if the former is run first, so rename it.